Automattic/pocket-casts-ios
# Onboarding Updates: Adds the updated Login landing view design

| 📘 Project: #500 | 
|:---:|

I apologize for the size of this PR, a lot of it is image assets that were added. 

This adds the updated login landing design with some basic navigation to the sign up / create account view:
- I also added a `EventDelegateHostingController` to allow a coordinator to react to the view controller events
    - This was needed because the the LoginCoordinator to interact with both the SwiftUI view and the hosting controller
    - I am using it to set the navigation bar style, because when I tried doing this through SwiftUI the changes didn't take effect
- I added a MotionManager class to easily interact with the device motion, I'm using this for a subtle parallax effect
- For fun I added an tap handler that makes the podcast image bounce
- I added an EnvironmentValue for `appTheme` that is setup in the `setupDefaultEnvironment`. This was done because iOS 14 ButtonStyle's doesn't get the EnvironmentObject passed to it, but it does take the custom environment value.
- I moved some assets from the single sign on feature branch into this one

## Screenshots

| iPhone SE 14.1 | iPhone 8  - 15.5 | iPhone 14 Pro Max - 16.1 |
|:---:|:---:|:---:|
|<img width="200" src="https://user-images.githubusercontent.com/793774/201232022-9e189ce2-3a21-4031-926e-d64f6a31c313.png">|<img width="200" src="https://user-images.githubusercontent.com/793774/201232013-e8d86e97-5de8-4237-b676-f72bf603a7e1.png">|<img width="200" src="https://user-images.githubusercontent.com/793774/201232030-4b4376b6-126c-45b3-a347-5a91e8316a59.png">|

## To test

1. Launch the app with the onboardingUpdates Feature Flag enabled
2. Go to the Profile screen
3. Tap the Create Folder button
4. ✅ Verify the view looks good on multiple screen sizes, and themes
5. ✅ Verify the view [matches the designs](https://www.figma.com/file/PviUP0aiZctOprDuuYRwpb/Onboarding-%26-Upgrade-v2?node-id=305%3A21965)
6. Tap the Sign Up button
7. ✅ Verify you are brought to the new account view
8. Tap back
9. Tap the Log In button
10. ✅ Verify you are brought to the login view
11. Tap the Not Now button
12. ✅ Verify the view dismisses 


## Checklist
